tee-worker/local-setup/launch.py

```diff
@@ -79,6 +80,11 @@ def is_port_open(port):
     except OSError:
         return False
 
+def is_directory_present(directory_path):
```

Comment:
isn't this function redundant ? 
It only proxies calls to `os.path.exists`.

Reply:
It is

Reply:
Not using this as always creating new_dir each time